EventManagement-Backend/src/main/java/com/group/eventmanagement/service/UserService.java

```diff
@@ -0,0 +1,70 @@
+package com.group.eventmanagement.service;
+
+import java.sql.Timestamp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.group.eventmanagement.model.User;
+import com.group.eventmanagement.repository.UserRepository;
+
+@Service
+public class UserService {
+
+	private UserRepository userRepository;
+	
+	@Autowired
+	public UserService(UserRepository userRepository) {
+		this.userRepository = userRepository;
+	}
+	
+	///// CREATION OF USER /////
+	@Transactional
+	public User createUser(String username, String password, 
```

Comment:
can you format method signature better

Reply:
not exactly sure what you mean, but I put everything on a single line(?)